### ISSUE TITLE
CPU 0% on convox instances

### ISSUE THREAD

**Issue by matthewford** [convox/rack#674]
```
convox instances
ID          AGENT  STATUS  STARTED     PS  CPU    MEM   
i-311414b6  on     active  2 days ago  1   0.00%  3.21% 
i-b8baac25  on     active  3 days ago  3   0.00%  38.47%
i-f67bdf6c  on     active  5 days ago  5   0.00%  42.47%

```

**Comment by nzoschke** [comment]
It is 0% because it is being calculated from the ECS CPU reservations, where none of the Convox tasks explicitly reserve CPU.

This is fixed in #703 by pulling data from CloudWatch Metrics.

**Comment by nzoschke** [comment]
Released on https://github.com/convox/rack/releases/tag/20160602143119. Thanks for the report!
